src/main/webapp/WEB-INF/resources/apps/ldt/translators/DMNTranslator.xsl

```diff
@@ -1,123 +1,384 @@
+				<xsl:apply-templates select="dmn:allowedValues"/>
+				<xsl:apply-templates select="dmn:itemComponent"/>
+			</dmno:ItemDefinition>
+		</dmno:itemComponent>
+	</xsl:template>
+	
+	<xsl:template match="dmn:literalExpression">
+		<xsl:param name="parentid"/>
+		<dmno:literalExpression>
+			<dmno:LiteralExpression rdf:about="{$parentid}_lE">
+				<xsl:apply-templates select="dmn:text"/>
+			</dmno:LiteralExpression>
+		</dmno:literalExpression>
+	</xsl:template>
+	
+	<xsl:template match="dmn:outputEntry">
+		<xsl:param name="dtid"/>
+		<xsl:param name="ruleNumber"/>
+		<dmno:outputEntry>
+			<dmno:LiteralExpression rdf:about="{$dtid}_rule_{$ruleNumber}_oE">
+				<dmno:relatedOutput rdf:resource="{$dtid}_output"/> <!-- Not part of the DMN specification -->
+				<xsl:apply-templates select="dmn:text"/>
+			</dmno:LiteralExpression>
+		</dmno:outputEntry>
+	</xsl:template>
+	
+	<xsl:template match="dmn:parameter">
+		<xsl:variable name="id">
+			<xsl:choose>
+				<xsl:when test="exists(@id)"><xsl:value-of select="@id"/></xsl:when>
+				<xsl:otherwise><xsl:value-of select="concat(translate(@name, ' ',''),'_p')"/></xsl:otherwise>
+			</xsl:choose>
+		</xsl:variable>
+		<dmno:parameter>
+			<dmno:InformationItem rdf:about="{$id}">
+				<rdfs:label><xsl:value-of select="@name"/></rdfs:label>
+			</dmno:InformationItem>
+		</dmno:parameter>
+	</xsl:template>
+	
+	<xsl:template match="dmn:variable">
+		<xsl:variable name="id">
+			<xsl:choose>
+				<xsl:when test="exists(@id)"><xsl:value-of select="@id"/></xsl:when>
+				<xsl:otherwise><xsl:value-of select="concat(translate(@name, ' ',''),'_v')"/></xsl:otherwise>
+			</xsl:choose>
+		</xsl:variable>
+		<dmno:variable>
+			<dmno:InformationItem rdf:about="{$id}">
+				<rdfs:label><xsl:value-of select="@name"/></rdfs:label>
+				<dmno:typeRef rdf:resource="{$feel-prefix}{substring-after(@typeRef,':')}"/>
+				<xsl:apply-templates select="dmn:description"/>
+			</dmno:InformationItem>
+		</dmno:variable>
+	</xsl:template>
+
+	<!-- Properties -->
+	<xsl:template match="dmn:description">
+		<dmno:description><xsl:value-of select="."/></dmno:description>
+	</xsl:template>
+	
+	<xsl:template match="dmn:informationRequirement|dmn:knowledgeRequirement|dmn:authorityRequirement">
+		<xsl:variable name="name"><xsl:value-of select="local-name()"/></xsl:variable>
+		<xsl:for-each select="*">
+			<xsl:element name="dmno:{$name}">
+				<xsl:attribute name="rdf:resource">urn:uuid:<xsl:value-of select="@href"/></xsl:attribute>
+			</xsl:element>
+		</xsl:for-each>
+	</xsl:template>
+	
+	<xsl:template match="dmn:namespace">
+		<dmno:namespace><xsl:value-of select="."/></dmno:namespace>
+	</xsl:template>
+	
+	<xsl:template match="dmn:owner">
+		<dmno:owner rdf:resource="urn:uuid:{@href}"/>
+	</xsl:template>
+	
+	<xsl:template match="dmn:text">
+		<dmno:text><xsl:value-of select="."/></dmno:text>
+	</xsl:template>
+
+	<xsl:template match="dmn:type">
+		<dmno:type><xsl:value-of select="."/></dmno:type>

```

Comment:
Is dit geen resource? In het originele model was dit feel:boolean, dus ik zou een vergelijkbare constructie verwachten als bij typeRef. Wat is overigens het verschil tussen type en typeRef??